code/modules/admin/fun_verbs.dm

```diff
@@ -1069,17 +1073,20 @@
 				return
 			if("Remove")
 				GLOB.offered_mob_list -= L
-				log_admin("[key_name(usr)] has removed offer of [key_name_admin(M)].")
-				message_admins("[ADMIN_TPMONTY(usr)] has removed offer of [ADMIN_TPMONTY(M)].")
+				log_admin("[key_name(usr)] has removed offer of [key_name_admin(L)].")
+				message_admins("[ADMIN_TPMONTY(usr)] has removed offer of [ADMIN_TPMONTY(L)].")
 				return
 
-	else if(alert("Are you sure?", "Offer Mob", "Yes", "No") != "Yes")
+	else if(alert("Are you sure you want to offer this mob?", "Offer Mob", "Yes", "No") != "Yes")
+		return
+
+	if(!istype(L))
```

Comment:
Maybe offer some feedback on why it failed, instead of silently erroring.

Reply:
Feel free to code it yourself

Reply:
No, this is a valid change request, please do it.

Reply:
```suggestion
	if(!istype(L))
		to_chat(usr,"<span class='warning'>Selected mob is no longer valid to offer.</span>")
```

Reply:
I could have just not included that and let it runtime lol